app/code/Magento/Analytics/Setup/InstallData.php

```diff
@@ -30,7 +30,7 @@ public function install(ModuleDataSetupInterface $setup, ModuleContextInterface
                     'scope' => 'default',
                     'scope_id' => 0,
                     'path' => 'analytics/subscription/enabled',
-                    'value' => 1
+                    'value' => 0
```

Comment:
I would suggest changing logic the validation, to better support the case with disabling validation, instead of disabling the feature on every new install. It was intended to be that way.

Reply:
@ishakhsuvarov I can do that, but then what industry should be set as a default? Otherwise, on fresh installs the reporting service will be enabled, but the **Industry** will be empty and "will cause distortion of the analytics reports", thus breaking the validation logic of `Magento\Analytics\Model\Config\Backend\Vertical::beforeSave()`. If the module is enabled by default and someone signs up for the service, will the Magento BI alert them that they need to select an **Industry**?

If that's handled, I can update the pull request so the module is enabled by default and update the disabling validation logic with:

```
// Magento\Analytics\Model\Config\Backend\Vertical
public function beforeSave()
{
    if (empty($this->getValue()) && ($this->getFieldsetDataValue('enabled') === '1')) {
        throw new LocalizedException(__('Please select an industry.'));
    }

    return $this;
}
```

which will allow disabling of the module, but make sure an **Industry** is supplied when trying to save the config with the module enabled.

Reply:
@nfourteen Currently waiting for someone from product organization to describe the desired state of the feature.
Thank you for collaboration

Reply:
@ishakhsuvarov Thanks for the update. Let me know when you hear something as I'd be happy to edit this PR to reflect the desired state.